tests/fixtures/chamber_of_deputies/reimbursements-with-comma

```diff
@@ -0,0 +1,2 @@
+congressperson_name,congressperson_id,congressperson_document,term,state,party,term_id,subquota_number,subquota_description,subquota_group_id,subquota_group_description,supplier,cnpj_cpf,document_number,document_type,issue_date,document_value,remark_value,net_value,month,year,installment,passenger,leg_of_the_trip,batch_number,reimbursement_number,reimbursement_value,applicant_id,document_id
+ABELARDO CAMARINHA,141463,329,2011,SP,PSB,54,1,Maintenance of office supporting parliamentary activity,0,,TIM CELULAR S/A,04206050005140,20272-AB,0,2009-05-19 00:00:00,411.51,0,411.51,5,2009,0,,,406387,2950,0,1772,1615640
```

Comment:
Is this file correct? The name says that the reimbursements has a comma, but the content has a value with a point.

Reply:
I renamed to reimbursements-with-decimal-point. Thank you for pointing out this.